### PR TITLE
Makes pizzas slip any individual unfortunate enough to step upon them

### PR DESCRIPTION
**Pull request by Paxilmaniac**
## About The Pull Request
As the title simply says, beware of floor pizzas.

## Why It's Good For The Game

https://user-images.githubusercontent.com/82386923/175401020-53a13e82-3970-4379-86e5-15bc62305a0d.mov

## Changelog

:cl:
add: Pizza will now slip any fool who steps upon it
/:cl:

### CODE REVIEW

**Comment by Paxilmaniac** [comment]
what th efuck is a gbp

File: code/game/objects/items/food/pizza.dm [hunk 1]
```diff
@@ -13,6 +13,10 @@
 	///What label pizza boxes use if this pizza spawns in them.
 	var/boxtag = ""
 
+/obj/item/food/pizza/ComponentInitialize()
+	. = ..()
+	AddComponent(/datum/component/slippery, 80)
```
[the remark this answers is not in the document]
Comment (Paxilmaniac) [reply code/game/objects/items/food/pizza.dm:18]: that would be a good idea I think

File: code/game/objects/items/food/pizza.dm [hunk 2]
```diff
@@ -13,6 +13,10 @@
 	///What label pizza boxes use if this pizza spawns in them.
 	var/boxtag = ""
 
+/obj/item/food/pizza/ComponentInitialize()
```
Comment (Fikou) [inline code/game/objects/items/food/pizza.dm:16]: componentinitialize is deprecated

Comment (Paxilmaniac) [reply code/game/objects/items/food/pizza.dm:16]: I am devastated by butterdog code lying to me

**Comment by optimumtact** [comment]
You currently have a negative Fix/Feature pull request delta of -7. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Pull request closed without merging**